crawler/plugins/applications/tomcat/tomcat_container_crawler.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+import dockercontainer
+from icrawl_plugin import IContainerCrawler
+from plugins.applications.tomcat import tomcat_crawler
+from utils.crawler_exceptions import CrawlError
+
+logger = logging.getLogger('crawlutils')
+
+
+class TomcatContainerCrawler(IContainerCrawler):
+    feature_type = 'application'
+    feature_key = 'tomcat'
+    default_port = 8080
+
+    def get_feature(self):
+        return self.feature_key
+
+    def crawl(self, container_id=None, **kwargs):
+        password = "password"
+        user = "tomcat"
+
+        if "password" in kwargs:
+            password = kwargs["password"]
+
+        if "user" in kwargs:
+            user = kwargs["user"]
+
+        c = dockercontainer.DockerContainer(container_id)
+
+        # check image name
+        if c.image_name.find(self.feature_key) == -1:
```

Comment:
Yes. If this "If" statement is removed, we can get tomcat's metrics correctly. But the overheads for containers and crawler become a little high because crawler tries to connect to all containers even if tomcat server does not run.  This if statement is also filtering function of the containers. I do not know which behavior with or without "if" statement is suitable for crawler.  Maybe, when the number of containers is small, this if statement is not needed. 
I am ready to delete and modify this statement if it is decided which behavior is suitable for crawler concept.